### PR TITLE
Dedicated Java Runtime View

### PR DESCRIPTION
**Pull request by akaroml**

### CODE REVIEW

File: src/java-runtime/index.ts [hunk 1]
```diff
@@ -9,6 +9,51 @@ import * as pathExists from "path-exists";
...
+
+  javaRuntimeView = vscode.window.createWebviewPanel("java.runtime", "Java Runtime", {
+    viewColumn: vscode.ViewColumn.One,
+  }, {
+    enableScripts: true,
+    enableCommandUris: true,
+    retainContextWhenHidden: true
+  });
+
+  await initializeJavaRuntimeView(context, javaRuntimeView, onDidDisposeWebviewPanel);
```
Comment (jdneo) [inline src/java-runtime/index.ts]: `javaRuntimeView` is defined at line 14, then why we still need to pass it as a method arg?

File: src/java-runtime/index.ts [hunk 2]
```diff
@@ -9,6 +9,51 @@ import * as pathExists from "path-exists";
...
+}
+
+function onDidDisposeWebviewPanel() {
+  javaRuntimeView = undefined;
```
Comment (jdneo) [inline src/java-runtime/index.ts]: Shall we call `javaRuntimeView.dispose()` before set it to `undefined`?

Comment (akaroml) [reply src/java-runtime/index.ts]: This event handler is called when the dispose() is called. So there's no need to call it again.

File: src/java-runtime/index.ts (hunk 1 above)
Comment (akaroml) [reply src/java-runtime/index.ts]: javaRuntimeView is defined outside but it does not stop the initializeJavaRuntimeView being implemented in a functional fashion. We try to only work with parameters and local variables.

**Pull request merged**